### PR TITLE
Journeys free tier updates

### PR DESCRIPTION
**Pull request by aeromusek**
Saw this float through as coming in today's deploy

@bptheo @zeeshabnc @Sarkar

### CODE REVIEW

File: pages/features/journeys.md [hunk 1]
```diff
@@ -57,43 +59,54 @@ If you have users in many countries, you can create a separate Journey for each
...
+{% endprotip %}
+
 ## Customize Template
 
-1. In the **Template Name:** field, enter a name to use for internal reference.
+1. In the **Customize Template** heading, you may edit the the name to use for internal reference.
 2. Click any object in the preview to edit it. To see documentation on all customization options, [click here](../advanced/#template-customization-options).
 3. When finished, click **Save & Close** button to continue.
 
+{% protip title="Deep linking into your app from Journeys" %}
```
Comment (aeromusek) [inline pages/features/journeys.md]: @mmolinet for you 😄

**Comment by Sarkar** [comment]
a heads up, the css editor has the ribbon now - perhaps update the screen?

**Comment by aeromusek** [comment]
Hey Journeys team...your new features have been live all weekend but your new docs are still waiting for approval!

Would be good to get these put through today :)

@bptheo @zeeshabnc @Sarkar

**Comment by zeeshabnc** [comment]
@aeromusek Can you update the CSS screen with the premium icon. 

Else lgtm, maybe wait on @bptheo thumb?

👍

**Comment by aeromusek** [comment]
@zeeshabnc can you clarify what you mean here? The documentation does currently have a premium star on the CSS editor section

**Comment by aeromusek** [comment]
@zeeshabnc aaaah I see you were referring to @Sarkar's comment. He and I chatted offline about that: he was referring to the screenshot of the CSS editor tab, which I took on a Premium-enabled account (where the star doesn't appear on the dashboard interface).

In general, I took the screenshots of premium features from the perspective of an account with a premium subscription. We both thought that made the most sense, though we could consider doing the reverse if you feel strongly.

**Comment by zeeshabnc** [comment]
ah ok, sounds good to me.

**Comment by aaustin** [comment]
it's just docs guys.. we don't have time for perfection.

**Pull request merged**